custom_model_runner/datarobot_drum/drum/gpu_predictors/__init__.py

```diff
@@ -85,6 +91,16 @@ def supported_payload_formats(self):
 
     def mlpiper_configure(self, params):
         super().mlpiper_configure(params)
+
+        self.status_reporter = MLOpsStatusReporter(
+            mlops_service_url=params["external_webserver_url"],
+            mlops_api_token=params["api_token"],
+            deployment_id=params["deployment_id"],
+            verify_ssl=self.verify_ssl,
+            # Events: download started|completed, server launching|ready
+            total_deployment_stages=4,
```

Comment:
You must not hard-code this. I don't think vLLM will have the same number of stages as Triton and NIM. This should be a class param or property i.e. `self.num_deployment_stages`

Reply:
I see you added the change here
https://github.com/datarobot/datarobot-user-models/blob/180035fc97dd3a5d297ffe1f1289db61027c07b4/custom_model_runner/datarobot_drum/drum/gpu_predictors/base.py#L110